uber/models.py

```diff
@@ -1247,6 +1395,27 @@ def _badge_adjustments(self):
     def _email_adjustment(self):
         self.email = normalize_email(self.email)
 
+    def _badge_adjustments(self):
```

Comment:
Also, for what it's worth, this is another bug that would have been caught by using `flake8` instead of `pep8`.